standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Text.Json.xml

```diff
@@ -1697,21 +1717,21 @@ The current depth exceeds the recursive limit set by the maximum depth.</excepti
     </member>
     <member name="M:System.Text.Json.Utf8JsonWriter.WriteCommentValue(System.ReadOnlySpan{System.Byte})">
       <summary>Writes a UTF-8 text value as a JSON comment.</summary>
-      <param name="utf8Value">The UTF-8 encoded value to be written as a JSON comment within /../.</param>
+      <param name="utf8Value">The UTF-8 encoded value to be written as a JSON comment within /*..*/.</param>
```

Comment:
I would be interested to see how VS picks this up now. Looks like back-ticks disappear (but at least /* is retained).